### DISCUSSION TITLE
Welcome to HSL.jl Discussions!

### DISCUSSION THREAD

**Discussion by dpo** [JuliaSmoothOptimizers/HSL.jl#84] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by farhadrclass** [comment]
@dpo 
Why do we only support HSL_jll = 2024, 2025 but not 2023?
